### DISCUSSION TITLE
Command To Open An Spesific Desktop App

### DISCUSSION THREAD

**Discussion by TeppenH** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#443] [Ideas]
Hello! I just tried this google assistant project. First I want to say that this project is amazing, but I think it would be great if you added a command to open the desktop application. So the model is like the google assistant on a smartphone. you just talk into the microphone or type "Ok Google, open File Explorer", then google assistant opens file explorer or "open Chrome", then google assistant can direct users to google chrome. 

so i think some command list like : 
- Open File Explorer
- Open Word / Powerpoint / Excel
- Open Chrome / Firefox / Edge / default browser
- Open Groove Music / Itunes / any music apps
- Open Whatsapp / Telegram / Signal / etc.
- Open Visual Studio Code / Sublime Text
- Open Zoom / Google Meet
- Open Gmail / Github / or any website (opened in chrome)
- etc.
  
That's my idea, hope it helps, thanks!

**Comment by sumeshir26** [comment]
Hi @TeppenH,
This feature has already been asked many times. Since this app is very much in beta, a lot of issues/enhancements need to be fixed before more advanced features like system integrated actions will be added afterwards.
As for the "Okay Google" request, it has already been added in the development version but has not been released yet but will be soon.
